### DISCUSSION TITLE
Compatibility with ledger (Income:Capital Gains)

### DISCUSSION THREAD

**Discussion by sohamshanbhag** [ananthakumaran/paisa#74] [Ideas]
Great app! I'm new to ledger and paisa seems great for viewing my finances.

I have a question regarding capital gains on selling, say, a Mutual Fund. In https://ledger-cli.org/doc/ledger3.html#Buying-and-Selling-Stock, the capital gains of a stock are supposed to be added to Income:Capital Gains. This would affect the taxation information in paisa.

Is there any possibility of supporting this? This helps when using ledger directly on the ledger file, without paisa, to find the capital gains.

**Comment by ananthakumaran** [comment]
Currently, paisa tries to calculate the price difference between buy and sell and use it to calculate xirr, etc. Adding capital gains would likely break those calculations. I will check the details and see how explicit capital gains entry can be supported

**Comment by ananthakumaran** [comment]
@sohamshanbhag This is supported on v0.5.6 https://paisa.fyi/reference/accounts/#capital-gains
